app/views/components/InputCoordinates.scala.html

```diff
@@ -0,0 +1,81 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.Coordinates
+@import viewModels.components.InputTextViewModel._
+@import views.html.components.InputText
+@import views.utils.ViewUtils._
+
+@this(
+    inputText: InputText,
+    govukFieldset: GovukFieldset
+)
+
+@(
+    form: Form[Coordinates],
+    messageKeyPrefix: String,
+    caption: Option[String],
+    headingArgs: Any*
+)(implicit messages: Messages)
+
+@html = {
+
+    <p class="govuk-hint">@messages(s"$messageKeyPrefix.hintText")</p>
+
+    @inputText(
+        field = form("latitude"),
+        inputTextType = TextInput(
+            label = messages(s"$messageKeyPrefix.latitude")
+        ),
+        autocomplete = Some("latitude"),
```

Comment:
To my knowledge, this is not a valid autocomplete parameter. Same goes for `longitude`. You should be looking to add an `a11y` spec for this new component and it will flag something like this.

Reply:
removed